.github/workflows/releaser.yml

```diff
@@ -10,10 +10,6 @@ on:
         description: 'Release tag'
         default: 'latest'
         type: string
-      checkout_ref:
-        description: 'Ref to checkout for doc index (default to current branch)'
-        default: ''
-        type: string
```

Comment:
You don't need this anymore in any case? @PProfizi

Reply:
Yes, that was for the old version of the action using Meilisearch.